### ISSUE TITLE
Add reader for zeiss dicom

### ISSUE THREAD

**Issue by marksgraham** [marksgraham/OCT-Converter#57]
Code for reading Zeiss' obfuscated dicom format is available here:

https://github.com/pydicom/pydicom/discussions/1618

I can implement but don't have any zeiss dicom data to test on - if anyone has any to share, let me know

**Comment by clairestreb** [comment]
@marksgraham , I have an actual file, so I will need to anonymize it first. However, I need to be able to see the images and make sure no PHI is in them (catch-22). I cannot run oct-converter. My python is rusty. Am I doing something wrong?
```
C:\OCT-Converter-master>python
Python 3.8.10 (default, Jul 16 2022, 12:16:26) [MSC v.1929 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> from oct_converter.readers import IMG
>>>
```

**Comment by marksgraham** [comment]
Implemented in PR #72
